project/plugins.sbt

```diff
@@ -6,6 +6,8 @@ resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
 
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
```

Comment:
It seems to me, that these changes are related to adding benchmarks, not the versions/plugins